Add note on RLS for Postgres guides (#212)
PeerDB-io/docs: snippets/rls-note.mdx

<Note>

> For initial load of the tables, the user must not be restricted by RLS policies. You can disable RLS policies for the user by running the below command:

Reviewer: RLS also applies to CDC's publication

> For initial load of the tables, the user must not be restricted by RLS policies. You can disable RLS policies for the user by running the below command:

Reviewer: ```suggestion
The user must not be restricted by RLS policies in order to replicate data. You can disable RLS policies for the user by running the below command:
```
RLS also applies to CDC's publication
